app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/MainActivity.java

```diff
@@ -229,6 +231,8 @@ public void uncaughtException(Thread paramThread, Throwable paramThrowable) {
         } catch (Exception e) {
             e.printStackTrace();
         }
+        audioManager = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
```

Comment:
If the audio is less than 50% then set it to 50%

Reply:
done